packages/base-manager/src/manager-base.ts

```diff
@@ -520,6 +554,45 @@ export abstract class ManagerBase implements IWidgetManager {
     moduleVersion: string
   ): Promise<typeof WidgetModel | typeof WidgetView>;
 
+  protected async loadModelClass(
+    className: string,
+    moduleName: string,
+    moduleVersion: string
+  ): Promise<typeof WidgetModel> {
+    try {
+      const promise: Promise<typeof WidgetModel> = this.loadClass(
+        className,
+        moduleName,
+        moduleVersion
+      ) as Promise<typeof WidgetModel>;
+      await promise;
+      return promise;
+    } catch (error) {
+      const msg = `Failed to load model class '${className}' from module '${moduleName}'`;
```

Comment:
Should we add a `console.log(error)` here as well? Like you did when failing to create a view.

Reply:
Added